src/compute-client/src/controller.rs

```diff
@@ -820,6 +821,7 @@ where
                 return Err(CollectionMissing(id));
             }
         }
+        let time_dependence = self.determine_time_dependence(instance_id, &dataflow)?;
```

Comment:
Should we just `expect` here? This can fail if the instance or one of the input collections is missing, both of which we have already excluded above.

Reply:
Yup; fixed.